test/lib/RuleManager.test.ts

```diff
@@ -148,4 +150,119 @@ describe('RuleManager', () => {
 			});
 		});
 	});
+
+	describe('runRulesMatchingCriteria()', () => {
```

Comment:
None of these tests call `runRulesMatchingCriteria`, though I assume that's what you're still working on.

Reply:
Oops #1. Yes, that is in progress.  Was just deciding where to put the tests, landed on mirroring getRulesMatchingCriteria

Reply:
Alright, I was able to add a simple test to run.test.ts as well as a few more here in RuleManager.test.ts.  I realized that I did not have to mock process.cwd().  I just needed to use process.chdir() to actually change it.  Duh.